backend/src/main/kotlin/app/ehrenamtskarte/backend/userdata/webservice/UserImportHandler.kt

```diff
@@ -37,15 +51,32 @@ class UserImportHandler {
 
             BufferedReader(InputStreamReader(file.content())).use { reader ->
                 getCSVParser(reader).use { csvParser ->
-                    importData(csvParser)
+                    importData(csvParser, project.id.value)
                 }
             }
-            ctx.status(200).result("Import successfully completed")
+            ctx.status(200).json(mapOf("message" to "Import successfully completed"))
         } catch (exception: UserImportException) {
-            ctx.status(400).result(exception.message ?: "Import failed")
+            ctx.status(400).json(mapOf("message" to exception.message))
+        } catch (exception: UnauthorizedException) {
+            ctx.status(401).json(mapOf("message" to exception.message))
+        } catch (exception: ForbiddenException) {
+            ctx.status(403).json(mapOf("message" to exception.message))
+        } catch (exception: ProjectNotFoundException) {
+            ctx.status(404).json(mapOf("message" to exception.message))
         } catch (exception: Exception) {
             logger.error("Failed to perform user import", exception)
-            ctx.status(500).result("Internal error occurred")
+            ctx.status(500).json(mapOf("message" to "Internal error occurred"))
+        }
+    }
+
+    private fun authenticate(ctx: Context): ApiTokenEntity {
```

Comment:
:upside_down_face: See comment before.
```suggestion
    private fun authenticate(context: Context): ApiTokenEntity {
```